ex4/VendingDP.Lib/MenuLoader.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace VendingDP.Lib
+{
+    public class MenuLoader
+    {
+        /* The logic of basic products and toppings is as follows: 
+            * they are held in a list of lists, each item of the outer list represents a product/topping.
+            * The inner lists are lists of [name, price, category], where the name will help identifying the ordered 
+            * item and the category will help associating toppings to a given basic product.
+            */
+        public static List<List<object>> basicProducts = new List<List<object>>();
```

Comment:
You should not use `object` at all. I saw you refer them as dictionaries. This pattern comes from python, but in OOP we try to model every object because it is a strongly typed language, such that we are able to catch errors in compile time.
Use what you learned: 
* Interfaces/abstract class - to hold different objects that have something in common
* Composition
* Inheritance 
And so on...

Reply:
Resolved